accounts/forms.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django import forms
+from django.contrib.auth import get_user_model
+
+
+def split_full_name(full_name):
```

Comment:
Either move this function to separate file "utils.py" or make this function a private function of related class.

Reply:
sure

Reply:
done